otsd

```diff
@@ -104,8 +101,12 @@ calendar_path = os.path.expanduser(args.calendar_path)
 calendar = otsserver.calendar.Calendar(calendar_path)
 aggregator = otsserver.calendar.Aggregator(calendar)
 
+proxy = bitcoin.rpc.Proxy()
+info = proxy.getinfo()
+relayfee = float(info['relayfee'])
+
 stamper = otsserver.stamper.Stamper(calendar,
-                                    args.btc_min_relay_feerate / 1000 * bitcoin.core.COIN,
+                                    relayfee / 1000 * bitcoin.core.COIN,
```

Comment:
Problem is, the relay fee isn't static and goes up and down over time.

There's a loop in the timestamp tx submission that immediately retries at a higher fee if the tx is rejected, so there's no need to get it right the first time (in fact I could probably remove the relayfee option entirely).